### ISSUE TITLE
Unknown function `nvim_create_buf`

### ISSUE THREAD

**Issue by neihc** [ncm2/float-preview.nvim#2]
I installed it by Vim Plug and got this error

```
Error detected while processing function <SNR>33_init:
line    6:
E117: Unknown function: nvim_create_buf
E15: Invalid expression: nvim_create_buf(0, 1)
line    7:
E121: Undefined variable: s:buf
E116: Invalid arguments for function nvim_buf_set_lines
line    8:
E121: Undefined variable: s:buf
E116: Invalid arguments for function nvim_buf_set_option
Press ENTER or type command to continue
```

My Neovim version is 0.3.1, 
I also tried in version 0.3.4

**Comment by roxma** [comment]
You should use neovim master.

Floating window feature has not been released yet.

**Comment by neihc** [comment]
I installed Neovim in master and it solved my issue
Thank you

**Comment by wookayin** [comment]
Floating window is available from 0.4.0 (nightly or pre-release). In practice, you can do sth like:

```
if has('nvim-0.4.0')
   Plug 'ncm2/float-preview.nvim'
endif
```
